cpp-client/include/nvidia/aiaa/model.h

```diff
@@ -63,6 +63,7 @@ struct AIAA_CLIENT_API Model {
   /// Image ROI size used while training [x,y,z,w] Format;  This shall be used for Annotation Models
   std::vector<int> roi;
 
+  /// Model Types support by AIAA
```

Comment:
should this be supported?
but this is not important though

Reply:
fixed